APITest/APITest/Controllers/BaseController.cs

```diff
@@ -15,5 +15,22 @@ protected async Task<object> GetAsync(string resource)
             var request = new RestRequest(resource, Method.GET);
             return await this.RestClient.GetAsync<object>(request);
         }
+        protected object Get(string resource)
+        {
+            var request = new RestRequest(resource, Method.GET);
+            return this.RestClient.Get<object>(request);
+        }
+
+        protected async Task<object> PostAsync(string resource)
+        {
+            var request = new RestRequest(resource, Method.POST);
+            return await this.RestClient.PostAsync<object>(request);
+        }
+
+        protected async Task<object> PutAsync(string resource)
+        {
+            var request = new RestRequest(resource, Method.PUT);
+            return await this.RestClient.PutAsync<object>(request);
+        }
     }
 }
```

Comment:
Created async methods for Post and Put requests and non-async for Get request (it was done for example)